website/docs/timeline.md

```diff
@@ -3,40 +3,386 @@ title: Timeline
 toc: true
 ---
 
-## Timeline
-At its core, Hudi maintains a `timeline` of all actions performed on the table at different `instants` of time that helps provide instantaneous views of the table,
-while also efficiently supporting retrieval of data in the order of arrival. A Hudi instant consists of the following components
+A Hudi table maintains all operations happened to the table in a single timeline comprised of two parts, an active timeline and an archived timeline. The active timeline stores all the recent instants, while the archived timeline stores the older instants. An instant is a transaction where all respective partitions within a base path have been successfully updated by either a writer or a table service. Instants that get older in the active timeline are moved to archived timeline at various times.
```

Comment:
```suggestion
A Hudi table records all operations performed on a table in a single timeline comprised of two parts, an active timeline and an archived timeline. The active timeline stores all the recent instants, while the archived timeline stores the older instants. An instant is a transaction where all respective partitions within a base path have been successfully updated by either a writer or a table service. Instants that get older in the active timeline are moved to archived timeline at various times.
```

Reply:
instant is not a transaction, just an id for a transaction. This file needs a deeper review still

Reply:
this is more or less the first concepts page. so feels early to discuss partitions and table services here.

Reply:
> Snuck in a few more comments. Happy to suggest edits to the top parts in the timeline doc

Please suggest changes.